examples/force_control/Force_control_example.py

```diff
@@ -0,0 +1,256 @@
+            if FT_sensor.sense()[2] > Fz_desired:
+                break
+        # step in simulation
+        world.step(sleep_dt=dt)
+    Fz_error_old = 0
+    sp_z = []
+    num = []
+    if flag == 1:
+        detx = np.array([0.0, 0.0, 0.0])
+    for t in count():
+        Fz_desired = 100
+        # move sphere
+        if t == 0:
+            z = robot.get_link_world_positions(link_id)[2]
+            sphere.position = np.array([0.46 - r * np.sin(w * t + np.pi / 2), r * np.cos(w * t + np.pi / 2), z])
+            # zz = z - 0.002  # Try to make the end-effector touch the surface of the table
+
+        # get current end-effector position and velocity in the task/operational space
+        x = robot.get_link_world_positions(link_id)
+        dx = robot.get_link_world_linear_velocities(link_id)
+        o = robot.get_link_world_orientations(link_id)
+        do = robot.get_link_world_angular_velocities(link_id)
+
+        # Get joint positions
+        q = robot.get_joint_positions()
+
+        # Get linear jacobian
+        if robot.has_floating_base():
+            J = robot.get_jacobian(link_id, q=q)[:, qIdx + 6]
+        else:
+            J = robot.get_jacobian(link_id, q=q)[:, qIdx]
+
+        # Pseudo-inverse: \hat{J} = J^T (JJ^T + k^2 I)^{-1}
+        Jp = robot.get_damped_least_squares_inverse(J, damping)
+        # Apply the admittance control
+        Fz_current = FT_sensor.sense()[2]  # record the current Fz
+
+        Fz_error = Fz_current - Fz_desired  # record the current error
+        # dv[2] = dv[2] + 0.00016 * Fz_error + 0.0000008 * (Fz_error - Fz_error_old) / dt # 结果较好的dt=2400
+        # dv[2] = 0.0013 * Fz_error + 0.0000020 * (Fz_error - Fz_error_old) / dt  # 结果较好的dt=2400
+        # dv[2] = 0.00093 * Fz_error + 0.000060 * (Fz_error - Fz_error_old) / dt
+        # dv[2] = 0.0052 * Fz_error
+        # sphere.position[2] = sphere.position[2] + 0.00095 * Fz_error + 0.000060 * (Fz_error - Fz_error_old) / dt
+        if flag == 0:
+            Fz_error_integral = Fz_error + Fz_error_old
+            zzz = sphere.position[2] + 0.000001 * Fz_error + 0.000002 * Fz_error_integral
+            Fz_error_old = Fz_error  # record the current error as the old error
+        elif flag == 1:
+            # xyz 3 direction impedance control
+            # M = np.array([[50, 0, 0], [0, 50, 0], [0, 0, 50]])
+            # D = np.array([[10, 0, 0], [0, 10, 0], [0, 0, 10]])
+            # K = np.array([[20, 0, 0], [0, 20, 0], [0, 0, 20]])
+            # numerator = np.array([[Fz_error[0], 0, 0], [0, Fz_error[1], 0], [0, 0, Fz_error[2]]]) * np.square(dt) \
+            #             + D * dt * dx[:, 1] + M * (2 * dx[:, 1] - dx[:, 2])
+            # denominator = M + D*dt + K*np.square(dt)
+            # dx_ = numerator * np.linalg.inv(denominator)
+            # dx[:, 2] = dx[:, 1]
+            # dx[:, 1] = dx[:, 0]
+            # dx[:, 0] = np.array([dx_[0, 0], dx_[1, 1], dx_[2, 2]])
+
+            M = 1
```

Comment:
May you provide some comments on how you came with the values for M (inertia), D (damping), and K (stiffness)? This info may be useful for future users